controllers/topics.controllers.js

```diff
@@ -0,0 +1,13 @@
+const express = require('express');
+const app = express();
+
+
+const topicData = require('../db/data/development-data/topics');
+
+const getTopics = app.get('/api/topics', (req, res) => {
+  const topics = topicData;
+  return res.status(200).send(topics);
+});
```

Comment:
This is fundamentally not what to do with a controller. This function should be invoking a model function that uses psql to interact with the database.